helm/charts/hpe-csi-driver/templates/hpe-csi-controller.yaml

```diff
@@ -64,6 +64,7 @@ spec:
           args:
             - "--v=5"
             - "--csi-address=$(ADDRESS)"
+            - "--timeoout=180"
```

Comment:
No, we don't want it to be hardcoded. it needs to be configurable like cspClientTimeout

Reply:
ok

Reply:
modified

Reply:
Have we tested this ? the timeout spelling is incorrect